shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -69,6 +69,12 @@ fn assert_query_result_contains_row(session: &Session, query: &str, row: &[Resul
     }
 }
 
+/// Execute a `query` against the `session` and assert the result rows does not contain `row`
+fn assert_query_result_not_contains_row(session: &Session, query: &str, row: &[ResultValue]) {
+    let result_rows = execute_query(session, query);
+    assert!(!result_rows.contains(&row.to_vec()));
```

Comment:
we need to include the row we unexpectedly found and every row we did find in the assert message.